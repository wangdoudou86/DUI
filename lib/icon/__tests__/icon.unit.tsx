import * as renderer from 'react-test-renderer';
import React from 'react';
import Icon from '../icon';

describe('Icon', () => {
    it('是一个svg图标', () => {
        const json = renderer.create(<Icon name="alipay"/>).toJSON()
        expect(json).toMatchSnapshot()
    })
}) 