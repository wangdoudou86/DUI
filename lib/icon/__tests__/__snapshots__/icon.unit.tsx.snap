// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon 是一个svg图标 1`] = `
<svg
  className="dui-icon"
>
  <use
    xlinkHref="#alipay"
  />
</svg>
`;
 